src/main/java/org/folio/circulation/domain/notice/TemplateContextUtil.java

```diff
@@ -92,6 +92,9 @@ public static JsonObject createCheckInContext(CheckInProcessRecords records) {
   }
 
   public static JsonObject createUserContext(User user) {
+    if (user == null) {
```

Comment:
Can the user ever be null? If so, in what context and how does the caller of this method handle it?

Reply:
Removed this check as unnecessary (now the condition is verified before this method).